### ISSUE TITLE
Accessibility kill option

### ISSUE THREAD

**Issue by tbektas0** [Fate-Grand-Automata/FGA#552]
I appreciate your contribution to fgo scripting community.
My suggestion is not fixing accessibility problem. I want to have option to forcefully close accessibility after exiting application. I have couple of example applications does that like auto clicker. 
After exiting app, application closes accessibility for itself. It would turn itself off.
 Right now, I need to force stop the app so it can turn off accessibility settings for next time. 
Currently, App gets stucks at accessibility on settings. 
Phone: Xiaomi mi 9T, miui optimisation is off.

**Comment by MathewSachin** [comment]
Having to turn ON Accessibility every time to use the app would be super annoying.
There's a pinned issue already for this Accessibility problem: #195. Does it not work for you?
I have confirmed with multiple people that those steps solve the problem.

**Comment by tbektas0** [comment]
Unfortunately it doesnt work me, it made my everytime I needed to restart my phone to now I need to force stop the app then turn on the accessibility. But I will try all of those solutions and come back again. My miui optimisation is off right now.

**Comment by reconman** [comment]
Are you asking for a feature where whenever you click on the `STOP SERVICE` button, accessibility is turned off?

**Comment by reconman** [comment]
In other words: Do you want to turn off accessibility after your AP have run out or do you have to turn it off and on again because of errors?

**Comment by tbektas0** [comment]
Yes I want to have stop service function so next time I can skip "force stop " step and directly open the app and do the rest of accessibility thing. Because stoping service doesnt turn off accessibility and it gets stuck at turned on for next time I open it it doesnt allow me to turned it on because it is already on.

Tldr : Stop service button should turn off accessibility for me without needing to force stop the app.

**Comment by reconman** [comment]
I didn't read the part about you having to do it after every restart. So your situation is "you have to turn it off and on again because of errors".

**Comment by tbektas0** [comment]
https://youtu.be/4K_VB34wQVA
